content/en/docs/contribute/intermediate.md

```diff
@@ -466,9 +476,23 @@ most up-to-date version of that branch.
     your changes applied. This is how reviewers will check your changes.
 
 9.  If you notice that more changes need to be made, or if reviewers give you
-    feedback, address the feedback locally, then repeat step 4 - 6 again,
-    creating a new commit. The new commit is added to your pull request and the
-    tests run again, including re-staging the Netlify staged site.
+    feedback, address the feedback locally and `amend` your original commit.
+
+    ```bash
+    git commit -a --amend
+    ```
+
+    - `-a`: commit all changes
+    - `--amend`: amend the previous commit, rather than creating a new one
+
+    An editor will open so you can update your commit message if necessary.
+
+    If you use `git commit -m` as in Step 4, you will create a new commit rather
+    than amending changes to your original commit. Creating a new commit means
```

Comment:
@aimeeu , I am not sure about recommending to amend commits. When reviewing PRs, I like to see how the PR evolved. When working locally, I sometimes will amend a commit before pushing to upstream. After the initial push, I prefer to see the subsequent commits. Also, PRs with multiple commits are accepted in k8s/website. I realize that squashing commits creates a clean commit history and is another task before merging.

Reply:
Thanks for the feeback, @kbhawkey. If we don't recommend amending the original commit, we again deviate from the Community's workflow. Also, It's my understanding that kubernetes/website _does_ require squashing multiple commits before a PR is merged, since that is the overarching policy for Kubernetes.  Squashing commits can be problematic, especially for those new to git/github. How about a compromise of recommending that with each amendment, the user does actually amend the commit message with what was done (ie, fix typos, enhance a section)?

Reply:
:wave: @aimeeu , It sounds like you are suggesting a new PR workflow for docs. Unless you enforce the use of amend or squash before merging, authors will do what they can/want. I still think it is OK to make mistakes when creating PRs, write content that requires a number of iterations to get right, and push a PR that is a WIP so that others can collaborate. I _think_ squashing is used throughout the k8s org over amend.

Reply:
@kbhawkey - I'm trying to reconcile the differences between the Community [workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md#5-commit) and the Docs workflow to reduce confusion.

![communityCommit](https://user-images.githubusercontent.com/26799583/66674350-4f9c4880-ec28-11e9-94e4-aba1db2fc326.png)


The Community workflow also has a section on squashing commits, which you don't need to do if you use `git commit --amend`. As to what contributors do in practice - I agree, people will do whatever they want. 

I'm beginning to think there isn't a single Docs PR workflow. Zach asked me to squash my commits on my first two "follow the Contributing doc directions" PRs, indicating that it was Kubernetes GitHub org policy to squash. 

What I'm trying to do is create the _simplest_ set of instructions for getting new contributors up and running as quickly as possible with the least amount of frustration. Most of us would agree that squashing, and any form of rebasing, is a level above basic git skills and can be very frustrating.   

Comparing `git commit` to `git commit --amend` - let's look at the steps involved when a new contributor submits a PR and updates it 12 times (not uncommon in the coding world, maybe less common in the Docs world). 

**Scenario git commit**

1.  new contributor submits a PR
2.  new contributor pushes 12 updates using `git commit` for each
3.  now there are 13 commits
4.  reviewers `/lgtm`
5.  an approver is assigned
6.  the approver says "squash your commits please so I can merge your PR"
7.  new contributor has to figure out how to squash commits

     - compare number of commits on Commit tab of UI to number of commits from `git log` - if they aren't the same, new contributor starts down the road to rebase hell and may end up abandoning the PR
     - go through the process of squashing the commits and possibly dealing with cryptic git messages

8.  new contributor has to push the squashed commits to the PR
9. ` /lgtm` removed because the bot detected another commit
10.  the approver  may not notice that the commits have been squashed unless the new contributor leaves a comment on the PR
11. the approver merges

**Scenario git commit --amend**
1. new contributor submits a PR
2. new contributor pushes 12 updates using `git commit --amend` for each
3. now there is still only 1 commit
4. reviewers `/lgtm`
5. an approver is assigned
6. the approver merges

Of the two scenarios, which is easier for the new contributor?

Reply:
As an aside, our Contribute docs don't cover how to create a good commit message, and by extension a good PR description. I've written guidelines for other projects ([example here](https://wiki.acumos.org/display/AC/Your+First+Patch#YourFirstPatch-WritingYourCommitMessage)] and will open an issue to update our docs.

Reply:
@aimeeu @kbhawkey I agree it's best to proceed with `git commit --amend`. While `git rebase` is useful knowledge, it's better educational design to build on a command we've already referenced, as opposed to a command we haven't yet taught or contextualized.